scripts/docker-integration-tests/query_fanout/m3coordinator-cluster-a.yml

```diff
@@ -58,3 +58,7 @@ carbon:
 
 tagOptions:
   idScheme: quoted
+
+query:
+  consolidation:
+    matchType: tags
```

Comment:
nit: Maybe add a comment to why it's valuable to test this configuration with query fanout?